docs/notebooks/Word2Vec_Multistream.ipynb

```diff
@@ -0,0 +1,212 @@
+    "\n",
+    "def processed_corpus():\n",
+    "    raw_corpus = api.load('wiki-english-20171001')\n",
+    "    for article in raw_corpus:\n",
+    "        doc = '\\n'.join(itertools.chain.from_iterable(zip(article['section_titles'], article['section_texts'])))\n",
+    "        yield preprocess_string(doc)        \n",
+    "\n",
+    "save_as_line_sentence(processed_corpus(), CORPUS_FILE)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Train Word2Vec\n",
+    "\n",
+    "We train two models:\n",
+    "* With `sentences` argument\n",
+    "* With `corpus_file` argument\n",
+    "\n",
+    "\n",
+    "Then, we compare timings and accuracy on `question-words.txt`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "from gensim.models.word2vec import Word2Vec, LineSentence\n",
+    "\n",
+    "st_time = time.time()\n",
+    "model_sent = Word2Vec(sentences=LineSentence(CORPUS_FILE), iter=5, size=300, workers=14)\n",
+    "model_sent_training_time = time.time() - st_time\n",
+    "\n",
+    "st_time = time.time()\n",
+    "model_corp_file = Word2Vec(corpus_file=CORPUS_FILE, iter=5, size=300, workers=14)\n",
+    "model_corp_file_training_time = time.time() - st_time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Training model with `sentences` took 9631.388\n",
+      "Training model with `corpus_file` took 4629.886 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Training model with `sentences` took {:.3f}\".format(model_sent_training_time))\n",
+    "print(\"Training model with `corpus_file` took {:.3f} seconds\".format(model_corp_file_training_time))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Training with `corpus_file` took two times less time!\n",
+    "\n",
+    "#### Now, let's compare the accuracies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gensim.test.utils import datapath"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/persiyanov/anaconda3/lib/python3.6/site-packages/gensim-3.5.0-py3.6-linux-x86_64.egg/gensim/matutils.py:737: FutureWarning: Conversion of the second argument of issubdtype from `int` to `np.signedinteger` is deprecated. In future, it will be treated as `np.int64 == np.dtype(int).type`.\n",
+      "  if np.issubdtype(vec.dtype, np.int):\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Word analogy accuracy with `sentences`: 0.753\n",
```

Comment:
One more test maybe (or that's enough?)

Reply:
which one?

Reply:
sorry, you already use exactly what should be